docs/source/technical_tutorials/working_with_r/access_aws_maap.ipynb

```diff
@@ -9,7 +9,7 @@
     "\n",
```

Comment:
<p>trivial: change <code>sep="/"</code> to <code>sep = "/"</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/476/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>